### PR TITLE
Add Junit tests for Activity List Command and Activity View Command

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

File: src/test/java/seedu/splitlah/command/ActivityListCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        Command createSession = Parser.getCommand(sessionArgs);
+        createSession.run(manager);
+        String activityOneArgs = "activity /create /sid 1 /n Lunch /p Alice /i Alice Bob Charlie /co 15";
+        Command createActivityOne = Parser.getCommand(activityOneArgs);
+        createActivityOne.run(manager);
+        String activityTwoArgs = "activity /create /sid 1 /n Lunch /p Alice /i Bob Charlie /cl 5 5";
+        Command createActivityTwo = Parser.getCommand(activityTwoArgs);
+        createActivityTwo.run(manager);
+    }
+
```
Comment (warrencxw) [inline src/test/java/seedu/splitlah/command/ActivityListCommandTest.java:30]: Consider removing this newline for standardisation (single line breaks only).

File: src/test/java/seedu/splitlah/command/ActivityListCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Checks if activities are listed with missing sid.
```
Comment (warrencxw) [inline src/test/java/seedu/splitlah/command/ActivityListCommandTest.java:33]: Consider rephrasing `sid` to `session unique identifier`.
Also consider splitting this test into two separate tests, one without the delimiter `/sid` (`activity /list 1`), one without the argument after `/sid` (`activity /list /sid`).

**Review by warrencxw — CHANGES_REQUESTED** [review]
Code is generally okay, just that there are minor standardisation issues to rectify and a simple suggestion from me.

File: src/test/java/seedu/splitlah/command/ActivityListCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        String activityTwoArgs = "activity /create /sid 1 /n Lunch /p Alice /i Bob Charlie /cl 5 5";
+        Command createActivityTwo = Parser.getCommand(activityTwoArgs);
+        createActivityTwo.run(manager);
+    }
+
+    /**
+     * Checks if activities are listed with missing session unique identifier.
+     */
+    @Test
+    public void prepare_hasMissingDelimiter_InvalidCommand() {
```
Comment (ivanaitzliddat) [inline src/test/java/seedu/splitlah/command/ActivityListCommandTest.java:35]: With regards to the other comment, can possibly split the two cases into two different methods.

**Review by ivanaitzliddat — CHANGES_REQUESTED** [review]
Looks good overall, but can probably refactor a little and change the naming for certain variables.

File: src/test/java/seedu/splitlah/command/ActivityListCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
```
Comment (froststein) [inline src/test/java/seedu/splitlah/command/ActivityListCommandTest.java:35]: Consider changing the function name to `prepare_hasMissingArguments_InvalidCommand`

File: src/test/java/seedu/splitlah/command/ActivityListCommandTest.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        String inputMissingSidDelimiter = "activity /list /sid";
+        Command activityWithMissingSidDelimiter = Parser.getCommand(inputMissingSidDelimiter);
+        assertEquals(InvalidCommand.class, activityWithMissingSidDelimiter.getClass());
+    }
+
+    /**
+     * Checks if activities are listed with missing argument after the delimiter.
+     */
+    @Test
+    public void prepare_hasMissingArgument_InvalidCommand() {
```
Comment (froststein) [inline src/test/java/seedu/splitlah/command/ActivityListCommandTest.java:46]: Consider changing the function name to `prepare_hasMissingDelimiter_InvalidCommand`

File: src/test/java/seedu/splitlah/command/ActivityViewCommandTest.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        String activityTwoArgs = "activity /create /sid 1 /n Lunch /p Alice /i Bob Charlie /cl 5 5";
+        Command createActivityTwo = Parser.getCommand(activityTwoArgs);
+        createActivityTwo.run(manager);
+    }
+    
+    /**
+     * Checks if activities are listed with missing delimiters.
+     */
+    @Test
+    public void prepare_hasMissingDelimiter_InvalidCommand() {
```
Comment (froststein) [inline src/test/java/seedu/splitlah/command/ActivityViewCommandTest.java:35]: Perhaps making it plural? with `hasMissingDelimiters`

**Review by froststein — APPROVED** [review]
Overall looks good to merge, just need to rename a few functions

**Review by ivanaitzliddat — APPROVED** [review]
Looks good to merge

**Pull request merged**